docs/space-daemon/getting-started.md

```diff
@@ -577,6 +577,29 @@ Returns a list of notifications objects. Notifications objects represent just sh
 
 ### Backup
 
+#### Toggle Bucket Backup
+
+> `[WIP]` <em>.toggleBucketBackup({ bucket: string, backup: boolean })</em>
+>
+> this method is still not supported by [space-daemon](https://github.com/FleekHQ/space-daemon)
```

Comment:
this could be removed since the `ToggleBucketBackup` method is already in the [space-daemon master branch](https://github.com/FleekHQ/space-daemon/blob/master/grpc/proto/space.proto#L265)

Reply:
I remove the WIP tag